src/NuGet.Clients/NuGet.Console/PowerConsoleCommands.cs

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Design;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.Shell;
+using NuGet.VisualStudio;
+using NuGetConsole.Implementation;
+using NuGetConsole.Implementation.PowerConsole;
+
+namespace NuGetConsole
+{
+    public class PowerConsoleCommands
+    {
+        private static readonly Lazy<PowerConsoleCommands> Lazy = new Lazy<PowerConsoleCommands>(() => new PowerConsoleCommands());
+        public static PowerConsoleCommands Instance => Lazy.Value;
+
+        private PowerConsoleWindow PowerConsoleWindow
+        {
+            get;
+            set;
+        }
+
+        private HostInfo ActiveHostInfo => PowerConsoleWindow.ActiveHostInfo;
+
+        private IWpfConsole WpfConsole
+        {
+            get;
+            set;
+        }
+
+        public async System.Threading.Tasks.Task InitializeAsync(AsyncPackage package)
+        {
+            await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync(package.DisposalToken);
+            var commandService = await package.GetServiceAsync(typeof(IMenuCommandService)) as OleMenuCommandService;
+            var componentModel = await package.GetComponentModelAsync();
```

Comment:
@dominoFire  @zivkan 
I'm going to remove this file and moving back content back to PowerConsoleToolWindow.cs
Because it's may be using GetServiceAsync but it's actually making to load a lot of unnecessary things with OleMenuCommandService( which only needed for PMC window load)  into memory at right start of the application, and it may not be needed at all if user doesn't do anything with PMC.